DESCRIPTION

```diff
@@ -43,6 +43,7 @@ Remotes: Displayr/flipTransformations,
     Tatvic/RGoogleAnalytics,
     cran/rgeolocate
 Suggests:
+    countrycode,
```

Comment:
Why is this in Suggests and not Imports?

Reply:
It adds another dep to open source users of the package. I think the r servers will install the suggests by default? So the r server will get it but open source users aren't required to get the dep?